libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -573,10 +573,6 @@ const cryptocurrenciesById: Record<string, CryptoCurrency> = {
       },
     ],
     explorerViews: [
```

Comment:
👇 Removing the explorer link to Hubble - it's a deprecated service.  `https://explorer.celo.org` is a fine default explorer.